### DISCUSSION TITLE
Figma source

### DISCUSSION THREAD

**Discussion by raivieiraadriano92** [heroui-inc/heroui#176] [Help] [answered]
Hey all
First I'd like to say that this project is amazing 🤩
I'd like to know if it would be possible to share the design source file as Figma file. If there is of course.
Thanks for the excellent work.

**Comment by jrgarciadev** [comment, marked as answer]
Hey @raivieiraadriano92 thank you so much, I'm glad you liked it 🙏🏻

Unfortunately, the NextUI Figma file is not public at the moment it will be available as soon as we release the first stable version

**Reply by raivieiraadriano92** [reply]
thanks 🙏🏻

**Reply by kilgharsap** [reply]
Do you have timeline on when it will be available? it will increase adaptability of this project

**Comment by Skor1k12214** [comment]
Отправлено с iPhone

**Comment by jrgarciadev** [comment]
https://www.figma.com/community/file/1267584376234254760/NextUI-Figma-Kit-(Community)

**Reply by muninn-huginn** [reply]
Hi, any estimation of when is it going to be updated?
